src/components/elements/alert/types.ts

```diff
@@ -1,15 +1,10 @@
 import { AlertProps, AlertType } from "@lifesg/react-design-system";
 import { TComponentOmitProps } from "../../frontend-engine";
-import type { IDividerSchema } from "../divider";
-import type { IOrderedListSchema, IUnorderedListSchema } from "../list";
-import type { ITextSchema } from "../text";
-import { IBaseElementSchema } from "../types";
+import { IBaseElementSchema, TBlockElementSchema, TInlineElementSchema } from "../types";
 
 export interface IAlertSchema<V = undefined, C = undefined>
 	extends IBaseElementSchema<"alert">,
 		TComponentOmitProps<AlertProps> {
 	type: AlertType;
-	children:
-		| React.ReactNode
-		| Record<string, IDividerSchema | IOrderedListSchema<V, C> | ITextSchema | IUnorderedListSchema<V, C>>;
+	children: React.ReactNode | Record<string, TBlockElementSchema<V, C> | TInlineElementSchema>;
```

Comment:
can we exclude
- `IAlertSchema`
- `IGridSchema<V, C>`
- `TWrapperSchema<V, C>`

Reply:
missed out on alert 🙇 

for grid and div, same as above, was thinking along the lines of layout in the alert

Reply:
got it, I will resolve this comment as the outcome for error-field typing will apply to alert typing